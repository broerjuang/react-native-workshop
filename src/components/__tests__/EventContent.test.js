import React from 'react';
import renderer from 'react-test-renderer';
import {EventContent} from '../index';

describe('Icon', () => {
  it('should render EventContent corectly', () => {
    let event = {
      profilePicture:
        'http://www.grosse.is-a-geek.com/robopics/roborovski01_1024.jpg',
      username: 'zzzcielo',
      action: 'COMMENT_ISSUE',
      actionTarget: 'Parallax Header Bug',
      repoTarget: 'astridtamara/bootcamp',
      date: '1d',
      comment:
        '[Test Long Comment] The bug is bugging me a lot. Good job fixing it in such short time.' +
        '[Test Long Comment] The bug is bugging me a lot. Good job fixing it in such short time.' +
        '[Test Long Comment] The bug is bugging me a lot. Good job fixing it in such short time.' +
        '[Test Long Comment] The bug is bugging me a lot. Good job fixing it in such short time. ',
    };
    let component = renderer.create(
      <EventContent event={event} openRepo={() => {}} openUser={() => {}} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
