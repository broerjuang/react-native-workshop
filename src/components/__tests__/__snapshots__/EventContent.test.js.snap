// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render EventContent corectly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    zzzcielo
     
  </Text>
  commented on issue
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
     
    Parallax Header Bug
     
  </Text>
  at
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
     
    astridtamara/bootcamp
     
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "grey",
      }
    }
  >
    1d
  </Text>
</Text>
`;
